sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -649,7 +650,35 @@ def monitoring_infos(self):
       for mi in op.monitoring_infos(transform_id).values():
         fixed_mi = self._fix_output_tags_monitoring_info(transform_id, mi)
         all_monitoring_infos_dict[monitoring_infos.to_key(fixed_mi)] = fixed_mi
-    return list(all_monitoring_infos_dict.values())
+
+    infosList = list(all_monitoring_infos_dict.values())
+
+    def inject_pcollection_into_element_count(monitoring_info):
+      if monitoring_info.urn == monitoring_infos.ELEMENT_COUNT_URN:
+        if not monitoring_infos.PTRANSFORM_LABEL in monitoring_info.labels:
+          return
+        ptransform_label = monitoring_info.labels[
+            monitoring_infos.PTRANSFORM_LABEL]
+        if not monitoring_infos.TAG_LABEL in monitoring_info.labels:
+          return
+        tag_label = monitoring_info.labels[monitoring_infos.TAG_LABEL]
+
+        if not ptransform_label in self.process_bundle_descriptor.transforms:
+          return
+        if not tag_label in self.process_bundle_descriptor.transforms[
+            ptransform_label].outputs:
+          return
+
+        pcollection_name = \
```

Comment:
Use brackets for multiline code instead of escaping newling with \
pcollection_name = (
    self.process_bundle_descriptor.transforms[ptransform_label].outputs[
              tag_label])

OR
pcollection_name = (self.process_bundle_descriptor.transforms[ptransform_label]
    .outputs[tag_label])